components/main-chef-wrapper/lib/version.go

```diff
@@ -0,0 +1,118 @@
+package lib
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"io/ioutil"
+	"encoding/json"
+	"github.com/mnogu/go-dig"
+	"github.com/chef/chef-workstation/components/main-chef-wrapper/dist"
+)
+
+
+
+var gemManifestMap map[string]interface{}
+var manifestMap map[string]interface{}
+
+func init() {
+	gemManifestMap = gemManifestHash()
+	manifestMap = manifestHash()
+}
+func Version(){
+	if omnibusInstall() == true {
+		showVersionViaVersionManifest()
+	} else {
+		// todo "5.1,0 is coming from #{ChefCLI::VERSION} in ruby, need to figure out how to do this "
```

Comment:
I think we can eliminate the branch that does not come from an omnibus install, because this wrapper is only supported in an omnibus install.